source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -117,7 +128,10 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
         button.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
         button.topAnchor.constraint(equalTo: topAnchor).isActive = true
         button.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
-        buttonLabelField.leadingAnchor.constraint(equalTo: button.trailingAnchor, constant: buttonConfig?.elementSpacing ?? 8).isActive = true
+        button.heightAnchor.constraint(greaterThanOrEqualToConstant: CGFloat(16)).isActive = true
+        button.widthAnchor.constraint(equalToConstant: CGFloat(16)).isActive = true
+        button.setContentCompressionResistancePriority(.required, for: .horizontal)
+        buttonLabelField.leadingAnchor.constraint(equalTo: button.trailingAnchor, constant: buttonConfig?.elementSpacing ?? 6).isActive = true
```

Comment:
Why was the constant changed from 8 to 6?
Any specific reason

Reply:
Constant 8 looks away from radio button, so make it look good by reducing space by 2 px. In addition, we have added 16 constants in width and height.

Reply:
Ok sure
Let's just not change in Webex app
As that is according to spec